src/main/webapp/app/oncokb-annotation-visualisation/AnnotationVisualisation.tsx

```diff
@@ -0,0 +1,380 @@
+  AnnotationVisualisationState
+> {
+  @computed
+  get mutationsAnnotations(): AnnotationImplication[] {
+    return this.props.annotations.filter(
+      annotation => annotation.alterationType === 'MUTATION'
+    );
+  }
+
+  @computed
+  get copyNumberAnnotations(): AnnotationImplication[] {
+    return this.props.annotations.filter(
+      annotation => annotation.alterationType === 'COPY_NUMBER_ALTERATION'
+    );
+  }
+
+  @computed
+  get structuralAnnotations(): AnnotationImplication[] {
+    return this.props.annotations.filter(
+      annotation => annotation.alterationType === 'STRUCTURAL_VARIANT'
+    );
+  }
+
+  @computed
+  get mutationsTreatments(): TreatmentImplication[] {
+    return this.props.treatments.filter(
+      treatment => treatment.alterationType === 'MUTATION'
+    );
+  }
+
+  @computed
+  get copyNumberTreatments(): TreatmentImplication[] {
+    return this.props.treatments.filter(
+      treatment => treatment.alterationType === 'COPY_NUMBER_ALTERATION'
+    );
+  }
+
+  @computed
+  get structuralTreatments(): TreatmentImplication[] {
+    return this.props.treatments.filter(
+      treatment => treatment.alterationType === 'STRUCTURAL_VARIANT'
+    );
+  }
+
+  @computed
+  get mutationsNotifications(): NotificationImplication[] {
+    return this.props.notifications.filter(
+      notification => notification.alterationType === 'MUTATION'
+    );
+  }
+
+  @computed
+  get copyNumberNotifications(): NotificationImplication[] {
+    return this.props.notifications.filter(
+      notification => notification.alterationType === 'COPY_NUMBER_ALTERATION'
+    );
+  }
+
+  @computed
+  get structuralNotifications(): NotificationImplication[] {
+    return this.props.notifications.filter(
+      notification => notification.alterationType === 'STRUCTURAL_VARIANT'
+    );
+  }
+
+  constructor(props: AnnotationVisualisationProps) {
+    super(props);
+    this.state = {
+      selectedAnnotationColumns: defaultAnnotationColumns,
+      selectedTreatmentColumns: defaultTreatmentColumns,
+    };
+  }
+
+  componentDidMount() {
+    const savedAnnotationColumns = localStorage.getItem(
+      'selectedAnnotationColumns'
+    );
+    const savedTreatmentColumns = localStorage.getItem(
+      'selectedTreatmentColumns'
+    );
+
+    if (savedAnnotationColumns && savedTreatmentColumns) {
+      this.setState({
+        selectedAnnotationColumns: JSON.parse(savedAnnotationColumns),
+        selectedTreatmentColumns: JSON.parse(savedTreatmentColumns),
+      });
+    }
+  }
+
+  handleAnnotationColumnsChange = (selectedOptions: any) => {
+    this.setState({
+      selectedAnnotationColumns: selectedOptions.map((option: any) =>
+        option.value ? option.value : option.prop
+      ),
+    });
+    localStorage.setItem(
+      'selectedAnnotationColumns',
+      JSON.stringify(selectedOptions.map((option: any) => option.value))
+    );
+  };
+
+  handleTreatmentColumnsChange = (selectedOptions: any) => {
```

Comment:
Can you use the type instead of `any`?

Reply:
This comment applies above as well and in most places you see `any` unless there is a good reason to use it